CHEWBBACA/CHEWBBACA_NS/AutoAlleleCuration.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+import os
+import argparse
+from Bio import SeqIO
+from Bio.Alphabet import generic_dna
+
+
+def main():
+
```

Comment:
For future proofing, I would add a comment with the point of this script, input type, expected output